cs-oracle-cloud/src/main/java/io/cloudslang/content/oracle/utils/Constants.java

```diff
@@ -0,0 +1,89 @@
+        public static final String BOOLEAN_FALSE = "false";
+        public static final String BOOLEAN_TRUE = "true";
+        public static final String STRICT = "strict";
+        public static final String EXCEPTION_NULL_EMPTY = "The %s can't be null or empty.";
+        public static final String EXCEPTION_INVALID_PROXY = "The %s is not a valid proxy details.";
+        public static final String EXCEPTION_INVALID_BOOLEAN = "The %s for %s input is not a valid boolean value.";
+        public static final String EXCEPTION_INVALID_NUMBER = "The %s for %s input is not a valid number value.";
+        public static final String EXCEPTION_INVALID_NAME = "The %s can only contain letters, numbers, underscores, " +
+                "and hyphens";
+        public static final String ANONYMOUS = "anonymous";
+        public static final String GET = "GET";
+        public static final String POST = "POST";
+        public static final String PATCH = "PATCH";
+        public static final String DELETE = "DELETE";
+        public static final String DEFAULT_JAVA_KEYSTORE = System.getProperty("java.home") + "/lib/security/cacerts";
+        public static final String CHANGEIT = "changeit";
+        public static final String ZERO = "0";
+        public static final String CONNECT_TIMEOUT_CONST = "10000";
+        public static final String UTF8 = "UTF-8";
+        public static final String CONNECTIONS_MAX_PER_ROUTE_CONST = "2";
+        public static final String CONNECTIONS_MAX_TOTAL_CONST = "20";
+        public static final String AUTHORIZATION = "Authorization:";
+        public static final String BEARER = "Bearer ";
+        public static final String OCI_HOST = "iaas.us-ashburn-1.oraclecloud.com";
+        public static final String LIST_INSTANCES = "/instances";
+        public static final String COMPARTMENT_ID = "compartmentId=";
+        public static final String PATH_SEPARATOR = "/";
+        public static final String AND = "&";
+        public static final String QUERY = "?";
+        public static final String ID = "id";
+        public static final String HTTPS = "https";
+        public static final String STATUS_CODE = "statusCode";
+        public static final String APPLICATION_VND_API_JSON = "application/vnd.api+json";
+        public static final String DELIMITER = ",";
+        public static final String DEFAULT_PAGE_NUMBER = "1";
+        public static final String DEFAULT_PAGE_SIZE = "100";
+        public static final String PAGE_NUMBER = "page[number]=";
+        public static final String PAGE_SIZE = "page[size]=";
+        public static final String SESSION_TIMEOUT = "20160";
+        public static final String SESSION_REMEMBER = "20160";
+        public static final String COLLABORATOR_AUTH_POLICY = "password";
+    }
+
+
+
+    public static class ListInstancesConstants {
+        public static final String LIST_INSTANCES_OPERATION_NAME = "List Organizations";
```

Comment:
```suggestion
        public static final String LIST_INSTANCES_OPERATION_NAME = "List Organizations";
``` 
Change the operation name or remove it